projects/module-2/whats-cookin-part-two-agile.md

```diff
@@ -0,0 +1,206 @@
+---
+title: What's Cookin - Part 2!
+tags: javascript, mocha, testing, fetch, agile
+---
+
+## Goals and Objectives
+
+- Work with a local server and make network requests to API endpoints to retrieve and manipulate data.
+- Ensure your app follows best practices for accessibility
+- Practice talking about your code and high level technical concepts
+- Implement feedback from a usability test to improve product
+- Improve group workflow.
+
+In this project, you will use your project from What’s Cookin’ (Part 1) and build on top of that. This is an opportunity to refactor your code, utilize agile processes, complete unfinished work, build new features, and take advantage of instructor feedback.
+
```

Comment:
It might be useful to add a short "why" paragraph, outlining that refactoring, testing, adding features, and implementing feedback are all part of dev work in the real world.